lib/http3/qpack.c

```diff
@@ -334,6 +334,11 @@ static int insert_literal_header(h2o_qpack_decoder_t *qpack, const char *name, s
     return decode_value_and_insert(qpack, header, value_is_huff, value, value_len, err_desc);
 }
 
+static int64_t qpack_table_cur(struct st_h2o_qpack_header_table_t *table)
+{
+    return table->base_offset + ((uintptr_t)table->last - (uintptr_t)table->first);
```

Comment:
Thank you for the PR. I like the direction of the PR but the cast to `(uintptr_t)` seems incorrect, PTAL #2724.